src/main/java/manageezpz/commons/core/Messages.java

```diff
@@ -10,7 +10,10 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT_BIND = "Invalid command format! \n\n%1$s";
 
     public static final String MESSAGE_EMPTY_NAME = "Name field cannot be empty! \n\n%1$s";
-
+    public static final String MESSAGE_EDIT_TASK_NO_EMPTY_VALUES =
+            "Either desc/ date/ or at/ must have a value after it!";
```

Comment:
Change this to

`"Either desc/ date/ or at/ must have a value after it! \n\n%1$s"`